src/main/java/bio/terra/cli/businessobject/WorkspaceUser.java

```diff
@@ -93,12 +94,12 @@ public static WorkspaceUser add(String email, Role role, Workspace workspace) {
   public static WorkspaceUser remove(String email, Role role, Workspace workspace) {
     // call WSM to remove a user + role from the current workspace
     WorkspaceManagerService.fromContext()
-        .removeIamRole(workspace.getId(), email, role.getWsmRole());
+        .removeIamRole(workspace.getUuid(), email, role.getWsmRole());
```

Comment:
This ship may have sailed, but could you make a separate PR for the automatic rename `id -> uuid` and kin? It's easier to approve them if the developer asserts it was an automatic refactoring.

Reply:
Excellent idea, will do next time :)